### DISCUSSION TITLE
Table row range select

### DISCUSSION THREAD

**Discussion by nik1a** [zauberzeug/nicegui#1593] [Ideas / Feature Requests]
### Question

Hi, is it possible to select a range of rows in a table using the shift+click?
Something like this https://codepen.io/coder-shmoder/pen/JLXVwr

**Comment by rodja** [comment]
Not out of the box but you may be able to code it yourself: [Capture the click events](https://github.com/zauberzeug/nicegui/discussions/979) and mark the rows accordingly. That would make [a great example](https://github.com/zauberzeug/nicegui/blob/main/CONTRIBUTING.md#examples) for NiceGUI if you get it working.

**Comment by falkoschindler** [comment]
I'll convert this question into a feature request, since it would be great to support range selection, either as an example or baked into `ui.table`.
Quasar has an example for [selection cell slots with range selection](https://quasar.dev/vue-components/table#example--selection-cell-slots-with-range-selection). Maybe we can borrow their code and move it into table.js. This could even be the default behavior for tables with multi selection, I guess.
